app/assets/stylesheets/components/_footer.scss

```diff
@@ -1,14 +1,15 @@
-
 .footer {
-  background: #F4F4F4;
+  background: #f4f4f4;
```

Comment:
I'd rather use the `$light-gray`as defined on the [config](https://github.com/Sajakhtar/cryptocanons/blob/master/app/assets/stylesheets/config/_colors.scss)

@khamzahabibullah 
@Sajakhtar

Reply:
Done, thanks!